test/framework/shootmigrationtest.go

```diff
@@ -99,13 +99,14 @@ func (t *ShootMigrationTest) MigrateShoot(ctx context.Context) error {
 	t.MigrationTime = metav1.Now()
 	return t.GardenerFramework.MigrateShoot(ctx, &t.Shoot, t.TargetSeed, func(shoot *gardencorev1beta1.Shoot) error {
 		t := gardencorev1beta1.Toleration{}
-		t.Key = core.SeedTaintProtected
+		t.Key = SeedTaintTestRun
+		t.Value = pointer.StringPtr("12345") //GetTestRunID())
```

Comment:
```suggestion
		t.Value = pointer.StringPtr("12345") //GetTestRunID())
```
what does the `GettestrunID` should the value be the id of the current testrun?
Can you add a comment to explain the value.

Reply:
`GetTestRunID` returns the current testmachinery testrun ID.
The value should match the taint that is set in the seed management test [here](https://github.com/gardener/gardener/blob/bcdf52d75e2df05c6ff2ee49524964a8c2b05302/test/framework/managedseedframework.go#L262)
And yes, it should be `GetTestRunID()`. I have forgot to revert it after my local tests